src/tools/gophish_export.py

```diff
@@ -126,7 +153,6 @@ def pull_gophish_campaign(api, assessment_id):
 
 def import_campaign(api, campaign_id):
```

Comment:
I would recommend renaming this function `get_campaign_data`.  

Similarly, I would rename `import_clicks` to `get_click_data`.

Reply:
Changed wording in fefb0b4.